project.godot

```diff
@@ -43,7 +43,11 @@ enabled=PackedStringArray("res://addons/godot-xr-tools/plugin.cfg")
 [physics]
 
 3d/run_on_separate_thread=true
-3d/physics_engine="GodotPhysics3D"
+3d/physics_engine="JoltPhysics3D"
+jolt_3d/sleep/enabled=false
+jolt_3d/solver/velocity_iterations=16
+jolt_3d/solver/position_iterations=6
+jolt_3d/solver/bounce_velocity_threshold=0.0
```

Comment:
No sleep: Otherwise stuff will just *stay* (put something on fridge -> open fridge -> the thing will just float through the lid)
bounce: Basically if we want any kind of bounce (yes, it's just a little but yes) this needs to be low. This is low.